vl-kontrakt-fp-tilkjent-ytelse/src/main/java/no/nav/foreldrepenger/kontrakter/tilkjentytelse/v1/TilkjentYtelseAndelV1.java

```diff
@@ -66,6 +66,13 @@ public static TilkjentYtelseAndelV1 tilBruker(Inntektskategori inntektskategori,
         return andel;
     }
 
+    public static TilkjentYtelseAndelV1 tilBruker(Inntektskategori inntektskategori, long beløp, SatsType satsType, String arbeidsgiverOrgNr) {
+        TilkjentYtelseAndelV1 andel = new TilkjentYtelseAndelV1(inntektskategori, beløp, satsType);
+        andel.utbetalesTilBruker = true;
+        andel.arbeidsgiverOrgNr = arbeidsgiverOrgNr;
```

Comment:
Vi trenger å sette arbeidsgiverAktørId også når det er satt. Hvis bruker har arbeidstaker andel hos en organisasjon blir arbeidsgiverOrgNr satt og hvis bruker har arbeidstaker andel hos en privat person som arbeidsgiver blir arbeidsgiverAktørId satt. Begge to kan ikke bli satt samtidig.